README.md

```diff
@@ -1,221 +1,172 @@
 
 _Graphical flow editor for Frank configurations._
 
 This project will help you visualize and edit your adapters in a flow format. The configurations should be in the “beautiful” syntax.
 
-The project has been tested in Mozilla Firefox and Google Chrome, other browsers will possibly give an inaccurate representation of the editor.
+The project has been tested in Mozilla Firefox and Google Chrome, other browsers will possibly give an inaccurate representation of the editor. If you'd like to report a bug, you can do so [here](https://github.com/ibissource/frank-flow/issues/new?assignees=&labels=bug&template=bug_report.md).
 
-## How to use the Frank!Flow
+## Features of the Frank!Flow
+
+First of all, the Frank!Flow consists of three 'modes' or 'views'. These are: Flow, Hybrid and Editor. Each of the modes focus on a specific use case, like editing visually or programatically. The modes are made with usability in mind, which means that the Flow is the most simple and the Editor is the most advanced.
 
 ### 📁 Explorer
 
-The explorer is presented on the left side of the window. The explorer will help you to browse configurations, add, rename and delete them. The file-tree is a representation of the file system where your configurations are located.
+The Explorer is presented on the left side of the application and can be shown or hidden with a toggle. The explorer will help you to browse configurations, add, rename and delete them. The Explorer is a representation of the file-system where your configurations are located.
 
-There are two buttons located above the file-tree. The first one is for adding a new configuration and the second one is to save the current selected configuration.
+There are some buttons associated with the Explorer that will perform actions on the file-system. Three buttons are located on the header and are used for adding, saving and modifying the current file (respectively). The modify button will open a modal in which the file can be renamed or deleted. The four buttons above the Explorer in Editor mode are used for adding, deleting and modifying a file as well as refreshing the Explorer (handy when files have changed on the file-system). A file or folder can be added to  by selecting the folder first then pressing the add button.
 
-You can select a configuration by clicking on it. The flow will be loaded on the white canvas. Two additional options will be provided by right-clicking a file: Rename and delete.
+You can select a configuration by clicking on it. The Flow will be loaded on the Canvas.
 
 ### 🎨 Palette
 
-The palette contains all the usable pipes and validators out of which the adapters are build. The pipes can be searched and are categorized respectively.
-
-The pipe will be added to the canvas by clicking on it.
-
-### 🔌 Flow
+The palette contains all the usable listeners, pipes, validators and an exit out of which the adapters are build. These elements can be searched and are categorized respectively.
```

Comment:
Two comments here:
- I do not see the validators. Can you explain how to add them, or omit the validators?
- What do you mean by categorizing elements? How can you do that?

Reply:
Ok.